clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisteredModels/RegisteredModelTableRow.tsx

```diff
@@ -69,20 +87,26 @@ const RegisteredModelTableRow: React.FC<RegisteredModelTableRowProps> = ({
         ]),
   ];
 
+  const handleModelNameNavigation = (rmId: string) =>
+    isArchiveRow
+      ? navigate(registeredModelArchiveDetailsUrl(rmId, preferredModelRegistry?.name))
+      : navigate(rmUrl);
+
+  const handleVersionNameNavigation = (mv: ModelVersion) =>
+    isArchiveRow
+      ? navigate(
+          archiveModelVersionDetailsUrl(mv.id, mv.registeredModelId, preferredModelRegistry?.name),
+        )
+      : navigate(modelVersionUrl(mv.id, mv.registeredModelId, preferredModelRegistry?.name));
+
   return (
     <Tr>
       <Td dataLabel="Model name">
         <div id="model-name" data-testid="model-name">
           <FlexItem>
-            <Link
-              to={
-                isArchiveRow
-                  ? registeredModelArchiveDetailsUrl(rm.id, preferredModelRegistry?.name)
-                  : rmUrl
-              }
-            >
-              <Truncate content={rm.name} />
-            </Link>
+            <Button variant="link" isInline onClick={() => handleModelNameNavigation(rm.id)}>
+              <Truncate content={rm.name} style={{ textDecoration: 'underline' }} />
```

Comment:
Looks like the underline style is not showing up in the truncated inline style button in MUI, not sure if it's because the PF version here is outdated.

Reply:
Just wanted to clarify - you mean the underline is not showing up for model name and version name? when I tested this locally, the underline appeared. so I just want to make sure I'm understanding your point correctly.
<img width="542" height="58" alt="Screenshot 2025-07-31 at 1 43 19 PM" src="https://github.com/user-attachments/assets/42644941-5c45-4c05-b3b9-b4ae34b9376a" />

Reply:
yeah, right. I will do that.